src/main/java/Parser.java

```diff
@@ -0,0 +1,20 @@
+public class Parser {
+    public static Command processCommand(String command) {
+        boolean isList = command.equals("list");
+        if(isList) return Command.LIST;
+
+        boolean isBye = command.equals("bye");
+        if(isBye) return Command.BYE;
+
+        boolean isDone = command.equals("done");
+        if(isDone) return Command.DONE;
+
+        boolean isDeadline = command.equals("deadline");
+        if(isDeadline) return Command.DEADLINE;
+
+        boolean isEvent = command.equals("event");
+        if(isEvent) return Command.EVENT;
+
+        return Command.TODO;
```

Comment:
For readability, it is better to put all the locally declared boolean variables together and then the if statement. I find it a bit confusing to look at the first glance.